src/tests/EdFi.DataManagementService.Tests.E2E/Features/SubclassResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,37 @@
+Feature: Subclass resources validatio
+
+        Background:
+            Given this Local Education Agency
+                  | localEducationAgencyId | nameOfInstitution | localEducationAgencyCategoryDescriptor | categories                 |
+                  | 255901                 | Grand Bend ISD    | ["Independent"]                        | ["Local Education Agency"] |
+
+        @ignore
+        Scenario: Ensure client can't get create a school with the same ID as another Subclass
+             When a POST request is made to "/ed-fi/schools" with
+                  """
+                  {
+                      "schoolId": 255901,
+                      "nameOfInstitution": "School Test",
+                      "educationOrganizationCategories": [
+                          {
+                              "educationOrganizationCategoryDescriptor": "uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#Educator Preparation Provider"
```

Comment:
Please pick a descriptor with a regular `uri://ed-fi.org` base URI instead of this one. The `tpdm` extension descriptors will never exist in our E2E tests, although they do exist in the ODS/API demonstration site. Best to just ignore anything TPDM related. (By the way, TPDM stands for Teacher Preparation Data Model. It is an extension for use by universities who are training future teachers).

Reply:
Changed, and understood